src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/53.diff.sql

```
@@ -0,0 +1,1223 @@
+
+        SELECT @origQuantitySearchParamRows = sum(row_count)
+            FROM sys.dm_db_partition_stats
+            WHERE object_Id = object_id('QuantitySearchParam')
+            AND index_id IN (0,1)
+
+        SELECT @origTokenNumberNumberCompositeSearchParamRows = sum(row_count)
+            FROM sys.dm_db_partition_stats
+            WHERE object_Id = object_id('TokenNumberNumberCompositeSearchParam')
+            AND index_id IN (0,1)
+
+        SELECT @origTokenQuantityCompositeSearchParamRows = sum(row_count)
+            FROM sys.dm_db_partition_stats
+            WHERE object_Id = object_id('TokenQuantityCompositeSearchParam')
+            AND index_id IN (0,1)
+
+        SELECT @tmpNumberSearchParamRows = sum(row_count)
+            FROM sys.dm_db_partition_stats
+            WHERE object_Id = object_id('Tmp_NumberSearchParam')
+            AND index_id IN (0,1)
+
+        SELECT @tmpQuantitySearchParamRows = sum(row_count)
+            FROM sys.dm_db_partition_stats
+            WHERE object_Id = object_id('Tmp_QuantitySearchParam')
+            AND index_id IN (0,1)
+
+        SELECT @tmpTokenNumberNumberCompositeSearchParamRows = sum(row_count)
+            FROM sys.dm_db_partition_stats
+            WHERE object_Id = object_id('Tmp_TokenNumberNumberCompositeSearchParam')
+            AND index_id IN (0,1)
+
+        SELECT @tmpTokenQuantityCompositeSearchParamRows = sum(row_count)
+            FROM sys.dm_db_partition_stats
+            WHERE object_Id = object_id('Tmp_TokenQuantityCompositeSearchParam')
+            AND index_id IN (0,1)
+
+    DECLARE @message varchar(100) = '';
+    IF (@origNumberSearchParamRows != @tmpNumberSearchParamRows)
+        BEGIN
+            SET @logTime = getUTCdate()
+            set @message = CONCAT(@origNumberSearchParamRows, @tmpNumberSearchParamRows,'NumberSearchParam rows does not match')
+            EXECUTE dbo.LogEvent @Process='Verification',@Status=@message,@Start=@logTime
+            RETURN;
```

>RETURN [](http://example.com/codeflow?start=12&length=6)

THROW here instead of return.

---

Fixed. Simple throw is giving me error, so added random error code, message and severity